<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportForm</class>
 <widget class="QWidget" name="ReportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчеты</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/main.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="registerLabel">
     <property name="text">
      <string>Реестр</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDateEdit" name="dateEdit">
     <property name="displayFormat">
      <string>MMMM.yyyy</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="countReportLabel">
     <property name="text">
      <string>Количество</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="registerBox"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="sumReportLabel">
     <property name="text">
      <string>Сумма</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="sumReportEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Отчет</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="reportButton">
     <property name="text">
      <string>Отчет</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="selectionLabel">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="countReportEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Отчет</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
